packages/twenty-front/src/modules/object-record/record-show/components/FieldsCard.tsx

```diff
@@ -82,7 +82,8 @@ export const FieldsCard = ({
       objectNameSingular !== CoreObjectNameSingular.Task &&
       fieldMetadataItem.name !== 'taskTargets',
   );
-  const isReadOnly = objectMetadataItem.isRemote;
+  const isReadOnly =
+    objectMetadataItem.isRemote || isDefined(recordFromStore?.deletedAt);
```

Comment:
let's introduce a util for that isRecordReadOnly, this will help keep the code maintainable